### ISSUE TITLE
More thaw failures

### ISSUE THREAD

**Issue by den1k** [juji-io/datalevin#23]
Running into an error similar to #13:

```
Execution error (ExceptionInfo) at datalevin.lmdb.LMDB/get_range (lmdb.clj:739).
Fail to get-range: "Thaw failed against type-id: 16"
clojure.lang.ExceptionInfo: Fail to get-range: "Thaw failed against type-id: 16" {:dbi "datalevin/eav", :k-range [:closed #object[datalevin.bits.Indexable 0x2f4befaf "datalevin.bits.Indexable@2f4befaf"] #object[datalevin.bits.Indexable 0x58644bae "datalevin.bits.Indexable@58644bae"]], :k-type :eavt, :v-type :long}
```

It only appears when I try to flatten a query, e.g.

this breaks
```
(d/q '[:find [(pull ?e [*]) ...] ; <--- difference
       :where [?e :block/refs _]]
  @conn)
```

but this

```
(d/q '[:find (pull ?e [*])
       :where [?e :block/refs _]]
     @conn)
```
works.

using `0.3.3`

**Comment by huahaiy** [comment]
I added a test for the "flattened" pull find spec, it works fine on my test data.

Could you please provide a minimal test case for this failure condition? For example, the schema, the sample data, so that I can reproduce and investigate. Thanks.

**Comment by huahaiy** [comment]
Released 0.3.4 to fix this. Thanks for reporting.

**Comment by den1k** [comment]
Just reading this. Glad you found it. Thank you!
